.github/fixtures/test-cli-arg-ignore-tags/commit.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+GIT_COMMITTER_DATE="2021-01-23 01:23:45" git commit --allow-empty -m "feat: add first beta feature"
+git tag v1.0.0-beta.1
+
+GIT_COMMITTER_DATE="2021-01-23 01:23:46" git commit --allow-empty -m "feat: add second beta feature"
+git tag v1.0.0-beta.2
+
+# WARNING: If we won't create this commit, 1.0.0 won't be created!
+#GIT_COMMITTER_DATE="2021-01-23 01:23:47" git commit --allow-empty -m "chore: why do i need a commit here?"
+git tag v1.0.0
```

Comment:
I guess we can wait for #707 before going ahead with this.

Reply:
Hmm, I guess that will take some time. We can update the fixtures with another valid case and move on I think. What do you think?

Reply:
Up to you. If you want I can uncomment this and push?

Reply:
Yup, please

Reply:
Done